emily/handler/tests/integration/withdrawal.rs

```diff
@@ -628,13 +617,14 @@ async fn update_withdrawals_is_forbidden(
 
     if is_forbidden {
         assert!(response.is_err());
+
         match response.unwrap_err() {
             testing_emily_client::apis::Error::ResponseError(ResponseContent {
                 status, ..
             }) => {
                 assert_eq!(status, 403);
             }
-            e => panic!("Expected a 403 error, got {e}"),
+            e => panic!("Expected a 403 error, got {:#?}", e),
```

Comment:
If you want to use the `Debug` implementation, then you can do:
```suggestion
            e => panic!("Expected a 403 error, got {e:#?}"),
```